check_telemetry.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# This script performs basic checks for telemetry script from Phase 0.
+# Based on parameters passed, it checks that:
+# * the telemetry file was created and has both valid instanceID and PRODUCT_FAMILY_XX (telemetry sending was enabled during run and was successful "-e" param)
+# * the telemetry file was not created (telemertry sending was disabled during run "-d" param)
+# * the telemetry file was created and has valid instanceID and no PRODUCT_FAMILY_XX (telemertry sending was enabled during run and was not successful "-u" param)
+
+
+set -e
+
+if [ "$#" -ne 2 ]; then
+  echo "This script requires 2 prameters:"
+  echo "* product parameter: ps, pxc, ppg, psmdb!"
+  echo "* result parameter: -e (enabled telemetry); -d (disabled telemetry); -u (unsuccessful telemetry) "
+  echo "Usage: ./package_check.sh <prod> <result>"
+  exit 1
+fi
+
+FILE_LOCATION="/usr/local/percona/telemetry_uuid"
+
+if [ $1 = "ps" ]; then
```

Comment:
Better to change this to if [ "$1" = "ps" ]
This will help in handling cases where they might contain spaces or special characters